components/x11/xf86-video-vbox/Makefile

```diff
@@ -0,0 +1,85 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Aurelien Larcher
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= xf86-video-vbox
+# Use version string from Xorg module
+COMPONENT_VERSION= 1.0.1
+COMPONENT_SUMMARY= xf86-video-vbox - VirtualBox UMS driver for the Xorg X server
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_GIT_HASH)
+COMPONENT_ARCHIVE= $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.xz
+COMPONENT_GIT_HASH=  ab07f97612d045d8c0a376174b39a294d028fef2
+COMPONENT_ARCHIVE_HASH= \
+  sha256:0b89efa6830fbf65b76482a98c06a95c027f49266725a7d4cf98e1a95b8e9467
+COMPONENT_ARCHIVE_URL= \
+  https://cgit.freedesktop.org/xorg/driver/xf86-video-vbox/snapshot/$(COMPONENT_NAME)-$(COMPONENT_GIT_HASH).tar.xz
+COMPONENT_PROJECT_URL = http://xorg.freedesktop.org
+COMPONENT_CLASSIFICATION= Drivers/Display
+COMPONENT_LICENSE = MIT
+COMPONENT_LICENSE_FILE = COPYING
+COMPONENT_FMRI = x11/server/xorg/driver/xorg-video-vboxvideo
```

Comment:
why not just x11/server/xorg/driver/xorg-video-vbox  (like x11/server/xorg/driver/xorg-video-ati)?

Reply:
The package name in configure.ac is xorg-video-vboxvideo and the convention seems to be xor-video-drivername: here drivername is vboxvideo (there is also vboxmouse).